packages/frontend/src/app/layout.tsx

```diff
@@ -15,8 +20,11 @@ export default function RootLayout({
   children: React.ReactNode
 }) {
   return (
-    <html lang="en">
-      <body className={inter.className}>{children}</body>
+    <html lang="en" className="h-full">
+      <body className={roboto.className + ' min-h-full bg-light'}>
```

Comment:
Is there a need to have `h-full`, if we have `min-h-full` in body?

Reply:
Yes, we need to have both of them, since html also doesn't have full height by default.